yt/visualization/particle_plots.py

```diff
@@ -261,8 +275,12 @@ def __init__(
             weight_field,
             field_parameters=field_parameters,
             data_source=data_source,
+            deposition=deposition,
+            density=density,
         )
 
+        self.projected = weight_field is None
```

Comment:
Unless this attribute already exists on the main branch, I'd prefer that the name better reflect its boolean type
```suggestion
        self.is_projected = weight_field is None
```

Reply:
This attribute already exists in `PlotWindow`

Reply:
indeed, then disregard !